prisma/schema.prisma

```diff
@@ -38,22 +38,17 @@ model users {
   role String
   title String?
   preferences userPreferences
+  siteSubjects siteSubjects[]
 }
 
 
 type userPreferences {
-  complete subjectsCompleted[]
-  star subjectsStarred[]
   sort Int? @default(0)
   config String?
```

Comment:
what are these remaining preferences used for?

Reply:
I can make the config property into a userDefaultConfiguration property for prisma